src/scss/components/_scroll-indicator.scss

```diff
@@ -1,37 +1,60 @@
 .scroll {
   background-color: $white;
-  display: block;
-  margin-bottom:76px;
+  margin-bottom: 76px;
   padding: 20px 0 0 0;
 
   &__button {
+    align-items: center;
     align-self: center;
     border: none;
     display: flex;
-    height: 40px;
-    margin:auto;
+    flex-direction: column;
+    margin: auto;
     width: 40px;
   }
 
-  &__button:focus {
```

Comment:
why eliminated this focus online: none?

Reply:
@caritonaranjo not having a button outline may represent an accessibility problem. Maybe some custom styles can be set to ensure it integrates well with de design.

Reply:
It is exactly the reason @jusk02 describe above. Accessibility mention the focus should be visible.